pkg/app/app_template_test.go

```diff
@@ -381,8 +382,9 @@ releases:
 		})
 	}
 
-	t.Run("fail due to known field", func(t *testing.T) {
+	t.Run("fail due to unknown field with goccy/go-yaml", func(t *testing.T) {
```

Comment:
You might also see this idiom, the same test is run `with goccy/goyaml` and `with yaml.v2`, so that we are sure that we don't break anything while migrating to goccy/goyaml or helmfile v1.